src/extension/background.js

```diff
@@ -576,6 +578,23 @@ const internalActions = {
     chrome.contextMenus.onClicked.addListener(async ({ menuItemId }, tab) => {
       if (!tab.url) return;
       internalActions[menuItemId](tab);
+      await chrome.scripting.executeScript({
+        target: { tabId: tab.id },
+        func: async () => {
+          try {
+            const sampleRUM = await import(chrome.runtime.getURL('rum.js'));
+
+            // Ensure window.hlx and window.hlx.sidekick exists
+            window.hlx = window.hlx || {};
+            window.hlx.sidekick = window.hlx.sidekick || { location: window.location };
+
+            const checkpoint = `sidekick:context-menu:${menuItemId}`;
```

Comment:
`menuItemId` is a variable from the service worker context, it will be unknown in the page context. But you can pass arguments to `func` using the `args` array. See [here](https://developer.chrome.com/docs/extensions/reference/api/scripting#properties_5) for the documentation and [here](https://developer.chrome.com/docs/extensions/reference/api/scripting#runtime_functions) for an example.